my_code.py

```diff
@@ -1,16 +1,17 @@
 # Collaborators (including web sites where you got help: (enter none if you didn't need help)
 #  
-x =input("name of a man ")
-y =input("verb for going somewhere ")
-z =input("adjective ")
-a =input("adjective ")
-b =input("verb ")
-c =input("adjective ")
-d =input("adjective ")
-e =input("adjective ")
-f =input("verb for going somewhere ")
+x = input("name of a man ")
+y = input("verb for going somewhere ")
+z = input("adjective ")
+a = input("adjective ")
+b = input("verb ")
+c = input("adjective ")
+d = input("adjective ")
+e = input("adjective ")
+f = input("verb for going somewhere ")
+
 print("There was a man named " + x + " who lived on a mountain in the middle of nowhere. Everyday," ,
 "he would go " + y + " to the lake on his mountain. The lake was very " + z + " and " + a + " , and " , 
```

Comment:
Be careful about combining strings - you can use + or comma, but not both between strings. Between lines 13 & 14 and between 16 and 17, you have a comma at the end a the line, followed by a + at the start of the next line so the strings are separated by ,+ which doesn't work. Pick the one you want and delete the other.